Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1294,7 +1294,7 @@ class BrowserViewController: UIViewController {
         if let homePanelController = homePanelController {
             return .HomePanels(homePanelState: homePanelController.homePanelState)
         }
-        guard let tab = tabManager.selectedTab where !tab.loading else {
+        guard let tab = tabManager.selectedTab else {
             return .Loading
```

Comment:
.Loading is indicating that the app is in a loading state and that there is no formal UI state (tab, home page, tab tray) yet determined. It should only be used when there is _no_ other state available.

Yes, isPrivate should be a property that is outside of any UI element, but that needs serious rethinking of the way that we deal with privacy in the app. This was all work that was going to be done in the future, but who knows when it will be looked at now.